src/main/kotlin/com/sourcegraph/common/ui/DumbAwareBGTAction.kt

```diff
@@ -21,3 +22,12 @@ abstract class DumbAwareBGTAction : DumbAwareAction {
       icon: Icon?
   ) : super(text, description, icon)
 }
+
+class SimpleDumbAwareBGTAction(
+    text: @NlsActions.ActionText String? = null,
+    private val action: () -> Unit
+) : DumbAwareAction(text) {
```

Comment:
How is that supposed to work? Shouldn't it inherit from `DumbAwareBGTAction`?

Reply:
Good catch!